### ISSUE TITLE
Companion displaying update available for wa that is not present ingame

### ISSUE THREAD

**Issue by Jodsderechte** [WeakAuras/WeakAuras-Companion#2193]
**Operating System:**
- [x] Windows
- [ ] Mac
- [ ] Linux

**Companion Version:**
```
5.1.2 (btw the hint in the issue template is wrong see below)
Check the bottom left of the "About" panel in the app.
```

**Companion App Console Log:**
```
userDataPath C:\Users\Jods\AppData\Roaming\weakauras-companion
index-e51aa852.js:74 buildAccountList
index-e51aa852.js:95 validateWowPath
index-e51aa852.js:95 buildVersionList
index-e51aa852.js:74 buildAccountList
index-e51aa852.js:96 Creating default directory
index-e51aa852.js:96 {"code":"ERR_NON_2XX_3XX_RESPONSE","timings":{"start":1701005038366,"socket":1701005038386,"lookup":1701005038391,"connect":1701005038426,"secureConnect":1701005038460,"upload":1701005038460,"response":1701005038610,"end":1701005038616,"phases":{"wait":20,"dns":5,"tcp":35,"tls":34,"request":0,"firstByte":150,"download":6,"total":250}},"name":"HTTPError","options":{"agent":{},"decompress":true,"timeout":{},"prefixUrl":"","body":"{\"ids\":[\"i5ymsgzMy\",\"crmKDRPPw\",\"N9tiB9mO2\",\"UfM0TP_PR\",\"c0XUQAYJq\",\"61G34UJ-W\",\"ndcoHm6Zl\",\"platerpowerbar\",\"Rcm39UnZV\",\"M8ejNxoo_\",\"PMdq2Rg_N\",\"UKpo0_GbE\",\"NameplateSpellCooldowns\",\"turHC0a5k\",\"mEN8Q6zUa\",\"RW4D58CEy\",\"N-Hb07L7A\",\"Ru_9FLVjI\"]}","ignoreInvalidCookies":false,"context":{},"hooks":{"init":[],"beforeRequest":[],"beforeError":[],"beforeRedirect":[],"beforeRetry":[],"afterResponse":[]},"followRedirect":true,"maxRedirects":10,"throwHttpErrors":true,"username":"","password":"","http2":false,"allowGetBody":false,"headers":{"user-agent":"got (https://github.com/sindresorhus/got)","content-type":"application/json","content-length":"244","accept":"application/json","accept-encoding":"gzip, deflate, br"},"methodRewriting":false,"retry":{"limit":2,"methods":["GET","PUT","HEAD","DELETE","OPTIONS","TRACE"],"statusCodes":[408,413,429,500,502,503,504,521,522,524],"errorCodes":["ETIMEDOUT","ECONNRESET","EADDRINUSE","ECONNREFUSED","EPIPE","ENOTFOUND","ENETUNREACH","EAI_AGAIN"],"backoffLimit":null,"noise":100},"method":"POST","cacheOptions":{},"https":{},"resolveBodyOnly":false,"isStream":false,"responseType":"json","url":"https://data.wago.io/api/check/","pagination":{"countLimit":null,"backoff":0,"requestLimit":10000,"stackAllItems":false},"setHost":true,"enableUnixSockets":true}}
index-e51aa852.js:96 promisesWagoCallsComplete
index-e51aa852.js:96 promisesWagoDataCallsComplete
index-e51aa852.js:96 no data received for DungeonAuras_BattleForAzeroth
index-e51aa852.js:96 no data received for DungeonAuras_Shadowlands
index-e51aa852.js:96 no data received for DungeonAuras_Dragonflight
index-e51aa852.js:96 no data received for CCTracker 
index-e51aa852.js:96 no data received for DungeonAuras_WarlordOfDraenor
index-e51aa852.js:96 no data received for DungeonAuras_Legion
index-e51aa852.js:96 no data received for DungeonAuras_MistOfPandaria
index-e51aa852.js:96 no data received for DungeonAuras_Affixes
index-e51aa852.js:96 no data received for DungeonAuras_Cataclysm
index-e51aa852.js:96 no data received for Ml_7ZkXep
index-e51aa852.js:96 no data received for i5ymsgzMy
index-e51aa852.js:96 no data received for crmKDRPPw
index-e51aa852.js:96 no data received for N9tiB9mO2
index-e51aa852.js:96 no data received for UfM0TP_PR
index-e51aa852.js:96 no data received for c0XUQAYJq
index-e51aa852.js:96 no data received for 61G34UJ-W
index-e51aa852.js:96 no data received for ndcoHm6Zl
index-e51aa852.js:96 no data received for platerpowerbar
index-e51aa852.js:96 no data received for Rcm39UnZV
index-e51aa852.js:96 no data received for M8ejNxoo_
index-e51aa852.js:96 no data received for PMdq2Rg_N
index-e51aa852.js:96 no data received for UKpo0_GbE
index-e51aa852.js:96 no data received for NameplateSpellCooldowns
index-e51aa852.js:96 no data received for turHC0a5k
index-e51aa852.js:96 no data received for mEN8Q6zUa
index-e51aa852.js:96 no data received for RW4D58CEy
index-e51aa852.js:96 no data received for N-Hb07L7A
index-e51aa852.js:96 no data received for Ru_9FLVjI
index-e51aa852.js:20 writeAddonData
index-e51aa852.js:40 WeakAuras.toc has version: 100200
index-e51aa852.js:19 Backup: Plater-_retail_#401511068#1-20231126123839.zip saved
index-e51aa852.js:19 Backup: WeakAuras-_retail_#401511068#1-20231126123839.zip saved

```

**WeakAuras Version:**
```

5.8.6
```

**WeakAuras Errors:**
```
None
```

**Describe the bug**
The companion app displays an aura that is no longer present ingame/in the fieles it tries to update and it is stuck doing so.

**To Reproduce**
Unsure how to reproduce you can try with my saved data.

**Expected behavior**
Nothing would be the correct behavior. Aka it not displaying it wanted to update in the companion.

**Screenshots**
![update available](https://github.com/WeakAuras/WeakAuras-Companion/assets/39654549/4d994340-7bcc-46f8-8a5f-492aefe0e065)

![companion update bugg](https://github.com/WeakAuras/WeakAuras-Companion/assets/39654549/dc3f1124-c17f-4bec-afee-6aa66eb071d0)

**Include a .zip file containing:**

[Companion.zip](https://github.com/WeakAuras/WeakAuras-Companion/files/13466905/Companion.zip)

Important note there is NOT an update pending ingame so it's just the companion display thats bugged

**Comment by cremor** [comment]
See #2152

**Comment by mrbuds** [comment]
This is fixed in version 5.1.3
